alpine/Dockerfile

```diff
@@ -27,4 +25,10 @@ ENV PATH $PATH:/scripts
 
 # Set tini as the default entrypoint
 ENTRYPOINT ["tini", "--"]
-CMD ["python"]
+# HACK: detect which Python we have and run it
+CMD ["/bin/sh", "-ec", \
```

Comment:
This is a bit of an annoying hack. Setting the `ENTRYPOINT` clears the existing `CMD` from the base image. It's not possible to use build arguments in `CMD`.

It may actually not be that bad, though, as in 99% of cases the `CMD` will be overridden by an image that builds on this one.